source/reference/method/MongoDBGridFSBucket-renameByName.txt

```diff
@@ -0,0 +1,62 @@
+=======================================
+MongoDB\\GridFS\\Bucket::renameByName()
+=======================================
+
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\GridFS\Bucket::renameByName()
+
+   Selects a GridFS file and any of its revisions by their ``filename``
+   and sets a new ``filename`` value.
```

Comment:
It confuses me that the sentence starts with "select". The main action is to change the name.

Reply:
I reworded to start with "Sets a new filename...", let me know if that reads better